python/ray/_private/import_thread.py

```diff
@@ -56,16 +58,9 @@ def _run(self):
         # import_pubsub_client.subscribe and before the call to
         # import_pubsub_client.listen will still be processed in the loop.
         import_pubsub_client.subscribe("__keyspace@0__:Exports")
-        # Keep track of the number of imports that we've imported.
-        num_imported = 0
 
         try:
-            # Get the exports that occurred before the call to subscribe.
-            export_keys = self.redis_client.lrange("Exports", 0, -1)
-            for key in export_keys:
-                num_imported += 1
-                self._process_key(key)
-
+            self._do_importing()
```

Comment:
Good refactoring. Is this necessary now?

Reply:
Yes, it is. This process might start later and it's a new worker. So it wants to load everything needed first. Otherwise, it'll hang there until there is a publish.